components/user/UserLink.js

```diff
@@ -24,15 +35,22 @@ export default function UserLink({
       href={`${BASE_URL}/api/profiles/${username}/links/${link._id}`}
       target="_blank"
       rel="noopener noreferrer"
-      className={`rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium dark:hover:bg-secondary-low/40 hover:bg-secondary-low/40 grow ${
+      className={`relative rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium ${ link.animation !== "Glow" ? "dark:hover:bg-secondary-low/40 hover:bg-secondary-low/40" : ""} grow ${
         isEnabled ? "" : "opacity-50"
-      }`}
+      } ${getLinkAnimation.get(link.animation)}`}
       style={{
         "--hover-color": colors[link.icon],
       }}
     >
-      <span style={{ color: colors[link.icon] }}>
-        <DisplayIcon aria-label={`${aria} icon`} />
+      <span class="relative">
+        <span style={{ color: colors[link.icon] }} className={`${getIconAnimation.get(link.animation)}`}>
```

Comment:
I think this doesn't need the backtics

```suggestion
        <span style={{ color: colors[link.icon] }} className={getIconAnimation.get(link.animation)}>
```